common/config/rush/pnpm-lock.yaml

```diff
@@ -1939,10 +1831,6 @@ packages:
       form-data: 3.0.1
     dev: false
 
-  /@types/node/10.17.13:
```

Comment:
curious about the removal of `@types/node/10.17.13`

Reply:
Good question... I'm honestly not entirely sure. Just taking what rush update gives me. Let me look into it a bit though